doc/strategyfiles.md

```diff
@@ -168,13 +168,16 @@ column_name: unique_email
 ```
 
 #### Column Strategy: `literal`
```

Comment:
literal is inserted into the query as-is, why does this need a type?

Reply:
You are right, I will remove it 😄